recipes-support/provisioning/esec-provisioning.bb

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021 Maik Otto <m.otto@phytec.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "Provisioning Tools for ESEC IOTDM Plattform"
```

Comment:
Some minor typing and grammar fixes for the summary: "Provisioning tools for ESEC IoT Device Manager platform"

Reply:
@motto-phytec s/plattform/platform/ :wink: